.github/workflows/docs.yml

```
@@ -4,7 +4,7 @@
 name: Docs
 
 on:
-  # pull_request:
+  pull_request:
```

Need to uncomment this line before merging:
```suggestion
  # pull_request:
```